integreat_cms/cms/models/media/directory.py

```diff
@@ -70,7 +76,8 @@ def search(cls, region, query):
         :rtype: ~django.db.models.query.QuerySet [ ~integreat_cms.cms.models.media.directory.Directory ]
         """
         return cls.objects.filter(
-            Q(region=region) | Q(region__isnull=True), Q(name__icontains=query)
+            Q(region=region, is_hidden=False) | Q(region__isnull=True),
```

Comment:
```suggestion
            Q(region=region) | Q(region__isnull=True, is_hidden=False),
```
Ah sorry, I'm a bit stupid: It's exactly the other way round: Either the directory is part of a region, then the value of the `id_hidden` field doesn't matter, or the directory is global (region is `None`), then we only want the directories which are not hidden :laughing: